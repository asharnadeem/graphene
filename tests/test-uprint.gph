int main(){
    int a = 1;
    float b = 2.0;
    string c = "3";
    node<string> n, o, p, q, r, s, t;
    n.id = 4;
    n.val = "5";
    n -> o;
    n -> p;
    n -> q;
    n -> r;
    n -> s;
    n -> t;
    print(a);
    print(b);
    print(c);
    print(n);
}