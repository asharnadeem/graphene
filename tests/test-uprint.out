1
2
3
id: 4, val: 5, edges: 6